templates/jlink/README.md

```diff
@@ -18,26 +20,31 @@ prepare UBI9 OpenJDK ImageStreams with `jlink-dev` support.
 
         cekit --descriptor ubi9-openjdk-17.yaml build docker
 
-2. Within your OpenShift project,
+2. Create an OpenShift project and namespace
+
+        oc new-project jlink-dev
+        oc create namespace jlink
+
+3. Within your OpenShift project,
 
-        oc create imagestream ubi9-openjdk-17
+        oc create imagestream openjdk-17-jlink-tech-preview
 
-3. You may need to configure your container engine to not TLS-verify the OpenShift
+4. You may need to configure your container engine to not TLS-verify the OpenShift
    registry. For Docker, add the following to `/etc/docker/daemon.json` and restart
    the daemon:
 
         {
           "insecure-registries": [ "default-route-openshift-image-registry.apps-crc.testing" ]
         }
 
-4. Log into the OpenShift registry, e.g.
+5. Log into the OpenShift registry, e.g.
 
         REGISTRY_AUTH_PREFERENCE=docker oc registry login
```

Comment:
Please reference podman rather than docker.

Reply:
the REGISTRY_AUTH_PREFERENCE trick is specific to docker. I'd like to keep it in, but I'm happy for the document to be default podman with exceptions for docker

Reply:
Similarly the bit above about insecure registries, the necessary configuration is different for podman. we should add that too

Reply:
…but let's track those asks as new JIRAs and not block this PR

Reply:
> …but let's track those asks as new JIRAs and not block this PR

Agreed.